.rubocop.yml

```diff
@@ -54,6 +54,16 @@ RSpec/EmptyLineAfterSubject:
   Exclude:
     - 'spec/factories.rb'
 
+# the Pundit RSpec DSL triggers this cop false positively
+RSpec/RepeatedExample:
+  Exclude:
+    - 'spec/policies/**/*'
+
+# the Pundit RSpec DSL triggers this cop false positively
+RSpec/RepeatedDescription:
+  Exclude:
+    - 'spec/policies/**/*'
+
```

Comment:
I'm OK with disabling these (at least for now, as [the Rubocop team have been working on this issue](https://github.com/rubocop-hq/rubocop-rspec/issues/333)) :+1:

Reply:
Thanks for digging into it, I've added a link to that PR here: a2f6ffd